src/rollup/index.ts

```diff
@@ -427,7 +434,15 @@ function normalizeOutputOptions(
 	if (mergedOptions.optionError) throw new Error(mergedOptions.optionError);
 
 	// now outputOptions is an array, but rollup.rollup API doesn't support arrays
-	const outputOptions = mergedOptions.outputOptions[0];
+	const mergedOutputOptions = mergedOptions.outputOptions[0];
+
+	const plugins = inputOptions.plugins;
+	inputOptions.plugins = Array.isArray(plugins)
+		? plugins.filter(Boolean)
+		: plugins
+		? [plugins]
+		: [];
+	const outputOptions = inputOptions.plugins.reduce(applyOutputOptionHook, mergedOutputOptions);
```

Comment:
This can be handled by the pluginDriver. To get access, you probably need to pass it to this function.